include/pokemon.h

```diff
@@ -507,8 +508,10 @@ struct BattleMove
     u32 parentalBondBanned:1;
     u32 skyBattleBanned:1;
     u32 sketchBanned:1;
-
-    u16 argument;
+    u32 rainAlwaysHit:1;
+    u32 hailAlwaysHit:1;
+    u32 snowAlwaysHit:1;
+    u32 sunAccuracy50:1;
```

Comment:
```suggestion
    u32 sunAccuracyHalved:1;
```
Slightly more verbose but reads more nicely and is consistent with the labels of the other flags.

Reply:
Thought about it, but that wording implies that 70 accuracy would go to 35 accuracy.

Reply:
Fair enough, it *is* hardcoded to 50 too and something like "sunAccuracyLowered" isn't descriptive enough considering that. Yuck. I hate Game Freak sometimes.